### ISSUE TITLE
Package has been renamed. install  `@vitest/eslint-plugin` to use latest updates. 

### ISSUE THREAD

**Issue by ST-DDT** [vitest-dev/eslint-plugin-vitest#537]
Currently, the npm package [`eslint-plugin-vitest`](https://www.npmjs.com/package/eslint-plugin-vitest) doesn't show that is now continued as a separate package [`@vitest/eslint-plugin`](https://www.npmjs.com/package/@vitest/eslint-plugin).

It would be nice if the message links/refers to the new package.

https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions

**Comment by veritem** [comment]
I lost access to my previous npm account. That's why this is released under vitest org.

**Comment by bruce-c-liu** [comment]
@veritem I don't think this should be closed. You may be able to reach out to NPM to get this resolved. Otherwise, you'll keep having confused folks like in #525. You'll also have people accidentally installing the wrong npm package.

**Comment by veritem** [comment]
@bruce-c-liu I have contacted NPM since June, but I haven't heard back from them. I have also tried to reach out to them publicly, but they didn't respond. https://x.com/veritemm/status/1821595278358544827

**Comment by bruce-c-liu** [comment]
Gotcha! Then I think that and pinning this issue is all you can do. Thanks!
